common/src/main/java/com/microsoft/hydralab/common/util/Const.java

```diff
@@ -153,7 +157,14 @@ interface StorageType {
     }
 
     interface StoragePropertyBean {
-        String LOCAL = "localProperty";
+        String LOCAL = "localStorageProperty";
         String AZURE = "azureBlobProperty";
     }
+
+    final class LocalStorageURL {
+        public static final List<String> API_PATH_PREFIX_LIST = List.of("/api/storage/local/upload", "/api/storage/local/download");
```

Comment:
```suggestion
        public static final List<String> API_PATH_PREFIX_LIST = List.of(CENTER_LOCAL_STORAGE_UPLOAD, CENTER_LOCAL_STORAGE_DOWNLOAD);
```

Reply:
Suggested by Le as well, but this usage is illegal here as both the list and the strings are constant.

Reply:
Then could you define an inner class to hold the list?